src/components/ConfigSelect/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useTranslation } from 'react-i18next';
+
+import Dropdown from '../Dropdown/Dropdown';
+
+import styles from './ConfigSelect.module.scss';
+
+import { getStoredConfig, setStoredConfig } from '#src/utils/configOverride';
+
+const configs = import.meta.glob('../../../test-e2e/data/*.json');
+
+const configOptions = Object.entries(configs).reduce<string[]>((acc, [pth]) => {
+  const parts = pth.split('/');
```

Comment:
It might be safest to wrap a try catch here in case files get put into that directory that aren't configs.